src/lib/type.ts

```diff
@@ -0,0 +1 @@
+export type Color = "White" | "Black";
```

Comment:
Capital letters - gross 🤮

Reply:
too bad. It saves me from doing work for the dialogs.